skip-form.js

```diff
@@ -1,6 +1,7 @@
 // Original Author:
 // https://github.com/rohitner/
 // Skips ERP annoying form that pops up
-window.setTimeout(() => {
-    document.getElementById("skiplink").click();
-}, 5000);
\ No newline at end of file
+// Pop Up removed from erp , commented for future Reference and Use
```

Comment:
Do not leave a comment, and remove the code altogether. We will have the code in GitHub history for future reference.